Sources/XMLCoder/Auxiliaries/XMLAttributeNode.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  XMLAttribute.swift
+//  XMLCoder
+//
+//  Created by Benjamin Wetherfield on 6/3/20.
+//
+
+public protocol XMLAttributeProtocol {}
+
+@propertyWrapper public struct XMLAttributeNode<Value>: XMLAttributeProtocol {
+    public var wrappedValue: Value
+
+    public init(_ wrappedValue: Value) {
+        self.wrappedValue = wrappedValue
+    }
+}
+
+extension XMLAttributeNode: Codable where Value: Codable {
+    public func encode(to encoder: Encoder) throws {
+        try wrappedValue.encode(to: encoder)
+    }
+
+    public init(from decoder: Decoder) throws {
+        try wrappedValue = .init(from: decoder)
+    }
+}
+
+extension XMLAttributeNode: Equatable where Value: Equatable {}
+extension XMLAttributeNode: Hashable where Value: Hashable {}
```

Comment:
Would need the same conditionals in the rest of the places where these are used, including tests.

```suggestion
extension XMLAttributeNode: Hashable where Value: Hashable {}
#endif
```